### DISCUSSION TITLE
Formating of .bib author entry

### DISCUSSION THREAD

**Discussion by Dylan-Ail-Mor** [retorquere/zotero-better-bibtex#2129] [Q&A] [answered]
Hello there.

I'm new to this so sorry if this is an obvious/basic question, but I can't seem to find an answer.

When I export to Better Biblatex from Zotero my .bib file shows the below:

`author = {Welsh Government}, date = {2019},`

but this creates a bibliography entry of:

`Government, W., 2019.`

Which isn't ideal... I suspect thisis a problem with RStudio/RMarkdown as using the Zotero Citation Style Editor with the same .csl file doesn't repeat the problem, but I seem to be able to fix the issue if I manually update the .bib file to either have double brackets around the author:

`author = {{Welsh Government}}, date = {2019},`

or to have a trailing comma:

`author = {Welsh Government,}, date = {2019},`

So... my question is... is there a way to export the .bib file from Zotero with either of those options? That way I don't have to fix the RStudio/Latex problem because I understand them even less!

Thanks!

**Comment by retorquere** [comment, marked as answer]
Switch the author name to single-field mode in zotero, it's a small button alongside the name.

**Reply by Dylan-Ail-Mor** [reply]
That'll work! Thanks!
